src/ctapipe/image/cleaning.py

```diff
@@ -459,6 +496,129 @@ def time_constrained_clean(
     return mask_core | mask_boundary
 
 
+def nsb_image_cleaning(
+    geom,
+    image,
+    arrival_times,
+    picture_thresh=8,
+    boundary_thresh=4,
+    min_number_picture_neighbors=2,
+    keep_isolated_pixels=False,
+    time_limit=None,
+    time_num_neighbors=1,
+    bright_cleaning_fraction=0.03,
+    bright_cleaning_threshold=None,
+    largest_island_only=False,
+    pedestal_factor=2.5,
+    pedestal_std=None,
+):
+    """
+    Clean an image in 5 Steps:
+
+    1) Get picture thresholds for `tailcuts_clean` in step 2) from interleaved
+        pedestal events `pedestal_std` is not None.
+    2) Apply tailcuts image cleaning algorithm - `ctapipe.image.cleaning.tailcuts_clean`.
+    3) Apply time_delta_cleaning algorithm -
+        `ctapipe.image.cleaning.apply_time_delta_cleaning` if time_limit is not None.
+    4) Apply bright_cleaning - `ctapipe.image.cleaning.bright_cleaning` if
+        bright_cleaning_threshold is not None.
+    5) Get only largest island - `ctapipe.image.morphology.largest_island` if
+        `largest_island_only` is set to true.
+
+    Parameters
+    ----------
+    geom: `ctapipe.instrument.CameraGeometry`
+        Camera geometry information
+    image: `np.ndarray`
+        pixel charges
+    arrival_times: `np.ndarray`
+        Pixel timing information
+    picture_thresh: `float` or `np.ndarray`
+        threshold above which all pixels are retained. Used for `tailcuts_clean`.
```

Comment:
You should note that this value is overridden in some cases. Later, if `pedestal_factor` is set, it sets the picture_thresh based on that value and this one.  That shoudld be explained, and maybe change the name so it's clear it's the nominal picture threshold?